apps/docs/src/app/core/component-docs/text/examples/text-max-lines.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'text-max-lines',
+    templateUrl: './text-max-lines.component.html'
+})
+export class TextMaxLinesComponent {
+    text = `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+    eiusmod tempor incididunt ut labore et dolore magna aliqua. Malesuada bibendum arcu vitae elementum. Ut etiam sit
+    amet nisl. Laoreet suspendisse interdum consectetur libero id faucibus nisl. Tellus in metus vulputate eu
+    scelerisque felis imperdiet proin fermentum. Ullamcorper morbi tincidunt ornare massa eget egestas purus viverra
+    accumsan. Mauris vitae ultricies leo integer malesuada nunc vel risus. Nisi est sit amet facilisis. Eu turpis
+    egestas pretium aenean. Nunc id cursus metus aliquam eleifend. Placerat in egestas erat imperdiet. Etiam dignissim
+    diam quis enim lobortis scelerisque. A erat nam at lectus. Amet nisl purus in mollis nunc sed id. Purus semper eget
+    duis at tellus at urna.`
+}
```

Comment:
Looking at source text there is no any `<b />`.
But inspecteing this in a browser (chrome) I see generated `<b />` elements.
Who generates it? It seems that it shoulf be working without these `b`s (at least in chrome)

![image](https://user-images.githubusercontent.com/3263744/107930907-48a3e980-6f84-11eb-9763-3ee22d25b3e4.png)

Reply:
This generated by [JavaScript Template Literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), and it's a default functionality.

For this example, there is no difference will it be a template literal or not.
I think implemented way has more readability, other one very very very long string.

Reply:
Oh, shame on me, did'nt know about this default transformation :)